pages/base.page.js

```diff
@@ -0,0 +1,45 @@
+const PAGE_TIMEOUT = 10000
+
+export default class BasePage {
+
+
+   /**
+    * Open url
+    * 
+    */
+   async open(ruta) {
+       await browser.url(ruta);
+   }
+
+
+   /**
+    * Wait for an element to be clickable
+    * @param {Object} element a clickear
+    */
+   async clickElement(element) {
+       await element.waitForClickable({ timeout: PAGE_TIMEOUT });
+       await element.click();
+   }
+
+
+   /**
+    * Send text method
+    * @param {Object} element que recibirá el texto
+    * @param {String} text a envíar 
+    */
+   async sendText(element, text){
+       await element.waitForClickable({ timeout: PAGE_TIMEOUT });
+              
+       await element.setValue(text);
+   }
+
+   /**
+    * Check expected error 
+    */
+    async checkExpectedError(ruta){
```

Comment:
This method doesn't do what its name suggest. It's navigating to a given route, not checking for errors. Consider removing it, as checking / validating is typically a responsibility for the tests, and is better implemented at that level (when defining the test). PageObjects' main responsibility is basically encapsulating a page and behaviour within their elements, but not specifying valid or invalid behaviour / state.

Reply:
deleted...